np5.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+def add_vector_to_matrix(vector, matrix):
+    """
+    Function that add a vector to each row of a given matrix
+    :return: matrix
+    """
+    for i in range(matrix.shape[0]):
+        matrix[i] += vector
```

Comment:
The beauty of numpy is that in most cases for loops are not needed, try using broadcasting.
https://numpy.org/doc/stable/user/basics.broadcasting.html